templates/includes/navigation.yaml

```diff
@@ -16,6 +16,14 @@ root:
 
 core:
   get-started:
+    installation-medias:
```

Comment:
Two of the pages in the nav are not on the front page (first and last nav item), and are only referenced in boards pages. Having them this way is a logical order and gives quick access to the installation-medias page, without cluttering the message on the front page, which is a kind of a "supported boards" page.